packages/wordpress-theme-boilerplate/lib/commands/create.js

```diff
@@ -158,21 +158,6 @@ async function create( command ) {
 
 	const { data: repo } = response;
 
-	// Update repo to disbale issues, projects and wiki.
```

Comment:
Wieso sollen Issues, Projects und Wiki nicht deaktiviert werden? Ich habe es extra eingebaut.

Reply:
@grappler Weil es auch ein Theme ohne darunterliegendes WordPress-Projekt-Repo geben kann. Dann sollten diese Sachen nicht deaktiviert werden.

Deaktivieren können wir sie automatisch, wenn es mal noch ein `wordpress-project-boilderplate` gibt.

Reply:
Meine Überlegungen als ich den Code geschrieben habe waren
- Wir erstellen öfters Themes für existierende Projekte als nur standalone Themes
- Es besteht immer noch die Möglichkeit Issues zu aktivieren wenn nötig
- Für mich ist es einfacher Issues zu aktivieren weil es mir fehlen als nach dem erstellen daran zu denken, dass ich die Issues, Wiki und Projects deaktiveren soll
- Wiki und Projects brauchen wir in sehr wenige Fälle

Wenn es für euch wirklich ein grossen Problem ist, dann würde ich lieber eine Option einbauen "Is standalone theme?". Bei True würde es die Issues, Projects und Wiki nicht deaktivieren.

Reply:
Es soll sich hierbei in erster Linie um ein Dev-Tool handeln und nicht um ein PM-Tool. Ob ein Repo Issues offen hat oder nicht ist nicht wirklich relevant für die Entwicklung.

Für solche Standard-Einstellungen gibt es andere Möglichkeiten, zB unser Projekt-Setup-Skript oder Probot.

Reply:
Ich bin nicht überzeugt. Ich finde die Default Einstellungen von GitHub nicht passend für uns 🤷🏻

Reply:
Wie gesagt, wir können dafür Probot verwenden: https://github.com/wearerequired/probot-settings

Reply:
Sieht dann so aus: https://github.com/wearerequired/wordpress-plugin-boilerplate/blob/master/.github/settings.yml